tests/oper.rs

```diff
@@ -707,7 +707,7 @@ fn gen_mat_vec_mul() {
         S2: Data<Elem = A>,
     {
         let ((m, _), k) = (lhs.dim(), rhs.dim());
-        reference_mat_mul(lhs, &rhs.to_owned().into_shape((k, 1)).unwrap())
+        reference_mat_mul(lhs, &rhs.as_standard_layout().into_shape((k, 1)).unwrap())
```

Comment:
Thanks, this change makes sense - it is a sign of `.to_owned()` improving and becoming more capable which is great. But we'll have to make a compatibility note or something.

Reply:
I added relevant instructions in the comment of to_owned. Maybe we should clarify the role of the return value of to_owned and improve it?

Reply:
to_owned's docs are fine - it's `into_shape` that's a bit tricky to use, and `to_owned` doesn't need to account for that 🙂